pkg/sql/sqlbase/structured.go

```diff
@@ -1868,6 +1868,30 @@ func (desc *TableDescriptor) validateCrossReferences(
 		}
 	}
 	// TODO(dan): Also validate SharedPrefixLen in the interleaves.
+
+	// Validate the all types present in the descriptor exist.
+	typeMap := make(map[ID]*TypeDescriptor)
```

Comment:
if we're not using the return value, any reason to not just do `util.FastIntSet`?
also add a comment the function is a wrapper to do caching?

Reply:
It's not useful to use `FastIntSet` for things like descriptor ID's, since they are usually larger than 64. Also, the call to `GetAllReferencedTypeIDs` needs the `TypeDescriptor`s.

Reply:
Added a comment